ssl/t1_lib.c

```diff
@@ -1318,7 +1318,7 @@ TICKET_RETURN tls_decrypt_ticket(SSL *s, const unsigned char *etick,
         /* Some additional consistency checks */
         if (p != sdec + slen || sess->session_id_length != 0) {
```

Comment:
Are you referring to this:
> 6.2.4 Storage durations of objects
> 1 An object has a storage duration that determines its lifetime. There are three storage durations: static, automatic, and allocated. Allocated storage is described in 7.20.3.
> 2 The lifetime of an object is the portion of program execution during which storage is
guaranteed to be reserved for it. An object exists, has a constant address, and retains
its last-stored value throughout its lifetime. If an object is referred to outside of its
lifetime, the behavior is undefined. The value of a pointer becomes indeterminate when
 the object it points to reaches the end of its lifetime.

I may be wrong, but the whole chapter talks about static and automatic memory.
Allocated memory is described in 7.20.3